ajax/var_type.js

```diff
@@ -0,0 +1,23 @@
+const log = console.log,
+      name = `sanket`;
```

Comment:
If you are going to have non interpolated strings then use single quotes for strings.